test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,54 @@
+package util.fixtures;
+
+import entities.Client;
+import enums.Roles;
+import enums.StatusEnum;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public final class ClientServiceFixtures {
+  public static final Long FAKEID = Long.valueOf(10L);
+
+  public static List<Client> buildClients(Integer size) {
+    List<Client> clients =
+        IntStream.range(0, size)
+            .mapToObj(i -> ClientServiceFixtures.buildClient())
+            .collect(Collectors.toList());
+    return clients;
+  }
+
+  public static Client buildClient() {
```

Comment:
Create another overload `buildClient(Long id)` that will do

```
var clientExample = new Client()
clientExample.setId(id);
Return buildClient(clientExample);
```

Use this in the mapToObj above to send index as the id

Reply:
solved